content-security-policy/generic/generic-0_10.sub.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
     <title>test implicit port number matching (requires port 80)</title>
-    <meta http-equiv="Content-Security-Policy content="script-src 'self' www.{{host}} 'unsafe-inline';">
+    <meta http-equiv="Content-Security-Policy content="script-src 'self' {{domains[www]}} 'unsafe-inline';">
```

Comment:
Could you modify the line below that build the www url to use the correct template?

http://www." + location.hostname => http://{{domains[www]}}

Reply:
The fact that this test passed before makes me suspect it's flawed entirely but I don't really have the cycles to investigate.

Reply:
Sure thing